src/dotnet/Chat.Service/Db/DbTextEntryAttachment.cs

```diff
@@ -22,20 +21,22 @@ public static string ComposeId(string chatId, long entryId, int index)
         => Invariant($"{chatId}:{entryId}:{index}");
 
     public TextEntryAttachment ToModel()
-        => new () {
-            ChatId = ChatId,
-            EntryId = EntryId,
+    {
+        var chatEntryId = new ParsedChatEntryId(ChatEntryId);
+        return new () {
+            ChatId = chatEntryId.ChatId,
```

Comment:
I think we can replace ChatId and EntryId properties with single ChatEntryId property in TextEntryAttachment. Can you check this? If we can replace, then apparently ParsedChatEntryId is no longer needed.

Reply:
since it affects many parts I would discuss it first and do it in a separate PR